openc3/data/config/item_modifiers.yaml

```diff
@@ -220,13 +220,3 @@ LIMITS_RESPONSE:
       description: Variable length number of options that will be passed to the
         class constructor
       values: .+
-KEY:
-  summary: Defines the key used to access this raw value in the packet.
-  description: Keys are often JsonPath or XPath strings
-  since: 5.0.10
-  parameters:
-    - name: Key string
-      required: true
-      description: The key to access this item
-      values: .+
-  example: KEY $.book.title
```

Comment:
Why was this removed? You still have an example in access_tlm.txt of KEY on an ITEM.

Reply:
It was just remove from item_modifiers because it was a dup (I think).  It is also in param_item_modifiers.

Reply:
Right and that's used by both ... not sure why it didn't get duplicated in the output though